packages/big-design/src/components/Select/spec.tsx

```diff
@@ -223,18 +222,6 @@ test('input button has aria-label', async () => {
   await waitForElement(() => getByRole('button'));
 });
 
-test('select menu opens when focused on input', async () => {
-  const { getByRole, getByTestId, queryByRole } = render(SelectMock);
-  const input = getByTestId('select');
-
-  expect(queryByRole('listbox')).toBeEmptyDOMElement();
-
-  fireEvent.focus(input);
-
-  expect(queryByRole('listbox')).not.toBeEmptyDOMElement();
-  await waitForElement(() => getByRole('option', { name: /mex/i }));
-});
-
```

Comment:
I think we should keep this one and just change wording to "click" and use `click()`.

Reply:
Good idea